invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/components/MembersSearchLayout.js

```diff
@@ -21,13 +22,27 @@ export class MembersSearchLayout extends Component {
     return (
       <>
         {/* auto column grid used instead of SUI grid for better searchbar width adjustment */}
-        <div className="auto-column-grid">
+        <div className="auto-column-grid three-columns">
           <div>
             <SearchBar fluid />
           </div>
+
           <div>
             <SearchFilters customFilters={customFilters} />
-            <Sort values={config.sortOptions} />
+          </div>
+
+          <div className="flex align-items-center">
+            <Sort
+              values={config.sortOptions}
+              label={(cmp) => (
+                <>
+                  <label className="mr-10 min-width-max-content">
+                    {i18next.t("Sort by")}
+                  </label>
+                  {cmp}
+                </>
+              )}
```

Comment:
I think it would be more efficient to export this component from invenio-search-ui package and reuse it